android/src/main/java/io/rownd/android/Rownd.kt

```diff
@@ -461,12 +489,31 @@ class RowndClient constructor(
             if (account.idToken == "") {
                 Log.w("Rownd", "Google sign-in failed: missing idToken")
             } else {
-                account.idToken?.let { idToken -> authRepo.getAccessToken(idToken, intent = googleSignInIntent, type = AuthRepo.AccessTokenType.google ) }
+                account.idToken?.let { idToken ->
+                    val tokenResp = authRepo.getAccessToken(idToken, intent = googleSignInIntent, type = AuthRepo.AccessTokenType.google)
+
+                    tokenResp?.let {
+                        eventEmitter.emit(RowndEvent(
+                            event = RowndEventType.SignInCompleted,
+                            data = buildJsonObject {
+                                put("method", RowndSignInType.Google.toString())
+                                put("user_type", it.userType.toString())
```

Comment:
not sure if this is the best channel to give feedback but this line (and other similar uses of the userType) will produce a String from the enum name ("NewUser" and "ExistingUser") while events dictionary in documentation gives `user_type: "new_user" | "existing_user"`.

Reply:
Thanks for pointing this out. This is an oversight/bug in the implementation and will be fixed in 2.9.1.